### PR TITLE
Apis 5711

### PR DESCRIPTION
**Pull request by robdawson-hmrc**

### CODE REVIEW

File: app/model/Application.scala [hunk 1]
```diff
@@ -284,13 +288,14 @@ case class SubscriptionNameAndVersion(name: String, version: String)
...
   type State = Value
-  val TESTING, PENDING_GATEKEEPER_APPROVAL, PENDING_REQUESTER_VERIFICATION, PRODUCTION = Value
+  val TESTING, PENDING_GATEKEEPER_APPROVAL, PENDING_REQUESTER_VERIFICATION, PRE_PRODUCTION, PRODUCTION = Value
   implicit val format = Json.formatEnum(State)
 
   val displayedState: State => String = {
     case TESTING => "Created"
     case PENDING_GATEKEEPER_APPROVAL => "Pending gatekeeper check"
     case PENDING_REQUESTER_VERIFICATION => "Pending submitter verification"
+    case PRE_PRODUCTION => "Active"
```
Comment (peteslater-ee) [inline app/model/Application.scala:298]: Why have we gone with Active for a pre-prod state?

Comment (robdawson-hmrc) [reply app/model/Application.scala:298]: I thought that from perspective of a gatekeeper user a pre-prod app is really the same as a prod app (it's been approved and the user has verified their email, the only difference is that the user hasn't clicked the 'I've done it' button on the Start Using Your Application page)

Comment (robdawson-hmrc) [reply app/model/Application.scala:298]: Good call I'll try to find out

Comment (robdawson-hmrc) [reply app/model/Application.scala:298]: Nabeel is happy with this approach I think we're ok

**Review by peteslater-ee — APPROVED** [review]

**Pull request merged**